### ISSUE TITLE
USER ISSUE: Cant change constitution

### ISSUE THREAD

**Issue by VseOdbornik** [StrangeLoopGames/EcoIssues#24709]
\
**User Id: (slg153495)**
**Version:** 0.10.1.2 beta release-584

**Steps to Reproduce:**
Adding constitution articles doesn't work
Revising anything in amendments makes the game crash

**Expected behavior:**
change the constitution

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60,00%>Exception: ArgumentException
Message:Object of type 'Eco.Gameplay.Civics.GameValues.GamePickerList' cannot be converted to type 'Eco.Gameplay.Civics.GameValues.GamePickerListAlias'.
Source:System.Private.CoreLib

System.ArgumentException: Object of type 'Eco.Gameplay.Civics.GameValues.GamePickerList' cannot be converted to type 'Eco.Gameplay.Civics.GameValues.GamePickerListAlias'.
   at System.RuntimeType.CheckValue(Object& value, ParameterCopyBackAction& copyBack, Binder binder, CultureInfo culture, BindingFlags invokeAttr)
   at System.Reflection.MethodBase.CheckArguments(Span`1 copyOfParameters, IntPtr* byrefParameters, Span`1 shouldCopyBack, ReadOnlySpan`1 parameters, RuntimeType[] sigTypes, Binder binder, CultureInfo culture, BindingFlags invokeAttr)
   at System.Reflection.Run</size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by D3nnis3n** [comment]
Can you provide the full logfile and the respective part of the logfile on the server?

**Comment by D3nnis3n** [comment]
We'll also need a savefile that has it happen.

**Comment by D3nnis3n** [comment]
Fixed in 10.1.3.
